### DISCUSSION TITLE
Support configuring the Nightingale left-side menu online

### DISCUSSION THREAD

**Discussion by bbaobelief** [ccfos/nightingale#1200] [Ideas]
有些公用dashboard，我想让所有人都能看见，现在没法提供统一入口，只能改前端代码加左侧菜单。
希望左侧菜单可以支持从数据库获取，不用每次都改代码了。
![image](https://user-images.githubusercontent.com/3918461/195828677-49987472-1d0c-46e0-a946-f3e88d7bfc61.png)

**Comment by UlricQin** [comment]
新版本支持设置监控大盘全员可见，后面可以弄个入口，展示所有这种全员可见的大盘
